<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditExecutablesDialog</class>
 <widget class="QDialog" name="EditExecutablesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify Executables</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="executablesListBox">
     <property name="toolTip">
      <string>List of configured executables</string>
     </property>
     <property name="whatsThis">
      <string>This is a list of your configured executables. Executables in grey are automatically recognised and can not be modified.</string>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="titleEdit">
       <property name="toolTip">
        <string>Name of the executable. This is only for display purposes.</string>
       </property>
       <property name="whatsThis">
        <string>Name of the executable. This is only for display purposes.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Binary</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="binaryEdit">
       <property name="toolTip">
        <string>Binary to run</string>
       </property>
       <property name="whatsThis">
        <string>Binary to run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="toolTip">
        <string>Browse filesystem</string>
       </property>
       <property name="whatsThis">
        <string>Browse filesystem for the executable to run.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Start in</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="workingDirEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseDirButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Arguments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="argumentsEdit">
       <property name="toolTip">
        <string>Arguments to pass to the application</string>
       </property>
       <property name="whatsThis">
        <string>Arguments to pass to the application</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="overwriteAppIDBox">
       <property name="toolTip">
        <string>Allow the Steam AppID to be used for this executable to be changed.</string>
       </property>
       <property name="whatsThis">
        <string>Allow the Steam AppID to be used for this executable to be changed.
Every game/tool distributed through Steam has a unique ID. MO needs to know this ID to start those programs directly, otherwise the program is started by steam and then MO will not work. By default, MO will use the AppID for the game.
Right now the only case I know of where this needs to be overwritten is for the Skyrim Creation Kit which has its own AppID. This overwrite is already preconfigured.</string>
       </property>
       <property name="text">
        <string>Overwrite Steam AppID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="appIDOverwriteEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Steam AppID to use for this executable that differs from the games AppID.</string>
       </property>
       <property name="whatsThis">
        <string>Steam AppID to use for this executable that differs from the games AppID.
Every game/tool distributed through Steam has a unique ID. MO needs to know this ID to start those programs directly, otherwise the program is started by steam and then MO will not work. By default, MO will use the AppID for the game (usually 72850).
Right now the only case I know of where this needs to be overwritten is for the Skyrim Creation Kit which has its own AppID (usually 202480). This overwrite is already preconfigured.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="closeCheckBox">
       <property name="toolTip">
        <string>If checked, MO will be closed once the specified executable is run.</string>
       </property>
       <property name="whatsThis">
        <string>If checked, MO will be closed once the specified executable is run.</string>
       </property>
       <property name="text">
        <string>Close MO when started</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="toolTip">
        <string>Add an executable</string>
       </property>
       <property name="whatsThis">
        <string>Add an executable</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/new/guiresources/resources/list-add.png</normaloff>:/new/guiresources/resources/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="toolTip">
        <string>Remove the selected executable</string>
       </property>
       <property name="whatsThis">
        <string>Remove the selected executable</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/new/guiresources/resources/list-remove.png</normaloff>:/new/guiresources/resources/list-remove.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditExecutablesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditExecutablesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
